src/components/experienceCard/ExperienceCard.js

```diff
@@ -15,6 +15,10 @@ export default function ExperienceCard({ cardInfo }) {
     return typeof values === "undefined" ? null : "rgb(" + values.join(', ') + ")";
   }
 
+  const GetSubPoints = ({ subPoints }) => {
```

Comment:
Could you rename `subPoints` to something like `desc-bullet` everywhere. It would seem more appropriate with the other values.